### PR TITLE
Added MCXRedirector and Mighty Mouse payloads

### PR DESCRIPTION
**Pull request by walkintom**

### CODE REVIEW

File: Manifests/ManagedPreferencesApple/com.apple.driver.AppleHIDMouse.plist [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+			</array>
+			<key>pfm_range_list_titles</key>
+			<array>
+				<string>Primary Button</string>
+				<string>Secondary Button</string>
+			</array>
+		</dict>
+		<dict>
+			<key>pfm_name</key>
+			<string>Button3</string>
```
[the remark this answers is not in the document]
Comment (kevinmcox) [reply Manifests/ManagedPreferencesApple/com.apple.driver.AppleHIDMouse.plist]: I could see "_making more than 1 button the primary button_" as a specific option someone would want to set to disable right-click completely though.

**Comment by walkintom** [comment]
@apizz Apologies for how long it's taken to review the changes you requested, I've tweaked the manifest so hopefully it's better now - please let me know. I've also added icons for both manifests.

**Comment by apizz** [comment]
@walkintom no worries.  Changes look OK, but it appears that you've done a couple more things that need to be addressed:

1. You've moved the com.apple.MCXRedirector to a different directory.  The ManifestsApple folder is meant for Apple-supported & documented payloads.  Because we've figured out this can be managed, but isn't documented or advertised by Apple, this should be in ManifestsPreferencesApple.

2. You've added MCXAirPort, which was not originally part of this PR

File: Manifests/ManifestsApple/com.apple.MCXAirPort.plist [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+		<string>system</string>
+		<string>user</string>
+	</array>
+	<key>pfm_platforms</key>
+	<array>
+		<string>macOS</string>
+	</array>
+	<key>pfm_description</key>
+	<string>AirPort settings</string>
+    <key>pfm_domain</key>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.MCXAirPort.plist]: Formatting seems slight off here

File: Manifests/ManifestsApple/com.apple.MCXAirPort.plist [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+			<key>pfm_description</key>
+			<string>This value describes the issuing organization of the profile, as displayed to the user</string>
+			<key>pfm_name</key>
+			<string>PayloadOrganization</string>
+			<key>pfm_title</key>
+			<string>Payload Organization</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.MCXAirPort.plist]: Extra space

File: Manifests/ManifestsApple/com.apple.MCXAirPort.plist [hunk 4]
```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://apple.com/macos</string>	
+	<key>pfm_documentation_url</key>
+	<string>https://developer.apple.com</string>
+	<key>pfm_targets</key>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.MCXAirPort.plist]: Looks like in your MCXRedirector you reordered prefs to be alphabetical, which is good.  Can we do the same here, w/ a `plutil -convert xml1`.  Should automatically reorganize.

**Comment by walkintom** [comment]
@apizz Yep, I'm currently trying to work out via #profilecreator-dev why the AirPort manifest ended up in this PR, it wasn't intentional, I had intended to start a new one for it. If you're happy for it to remain in this PR, I'll make the changes you requested.

**Comment by apizz** [comment]
Thank you.  It's likely because you're using your `master` branch.  Anything you commit there, even after you submit a PR, will get added.

**Comment by apizz** [comment]
Closed now that we're working with #184

**Pull request closed without merging**